### PR TITLE
FOR REVIEW: Overview updates, new terms for some pages, fixing links

### PR DESCRIPTION
**Pull request by astronautthis**
Staging links:
https://makerdao-1.gitbook.io/endgame-docs-staging/endgame/overview
Summary.md page also changed

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -6,18 +6,19 @@ The Endgame Plan is a proposal to overhaul and improve the governance and tokeno
 
 The Endgame Plan was [introduced](https://forum.makerdao.com/t/decentralized-voter-committee-on-tuesday-5pm-cest-all-mkr-holders-welcome/15166) by Rune Christensen, the co-founder of Maker. After a series of initial discussions, Maker Governance accepted the [MIP set](https://vote.makerdao.com/polling/QmTmS5Nf) ratifying this plan on 24 October 2022. 
 
-The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). Note that some parts are now out of date.  
+The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). The Endgame Constitution, covering the  proposed by Christensen is available [here](https://forum.makerdao.com/t/mip101-the-maker-constitution/19621). Note that some parts are now out of date.
```
Comment (LongForWisdom) [inline overview.md:9]: Seems like you're missing a word here? 'covering the proposed by Christensen'

File: overview.md [hunk 2]
```diff
@@ -6,18 +6,19 @@ The Endgame Plan is a proposal to overhaul and improve the governance and tokeno
...
+The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). The Endgame Constitution, covering the  proposed by Christensen is available [here](https://forum.makerdao.com/t/mip101-the-maker-constitution/19621). Note that some parts are now out of date.  
 
 
 {% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
 ## Goals
 
 The Endgame Plan has many goals which come together to create the Endgame State. A list of these goals is below:
+- Providing access to an Unbiased World Currency in the form of the decentralized stablecoin Dai.
+- Ensure that Maker has a meaningful charitable impact through financial success.
```
Comment (LongForWisdom) [inline overview.md:18]: Not sure this quite makes sense. financial success doesn't exactly equate to charitable impact. Probably just end at charitable impact?

File: overview.md [hunk 3]
```diff
@@ -26,6 +27,11 @@ The Endgame Plan has many goals which come together to create the Endgame State.
 
 The Endgame Plan consists of the following main components:
 
+### [The Maker Constitution](maker-core/constitution.md)
+The Maker Constitution specifies the core scope of Maker Governance. The Constitution is meant to be immutable as much as possible. This simplicity and immutability guarantees long-term transparency about the purpose of MakerDAO.
```
Comment (LongForWisdom) [inline overview.md:31]: 'As immutable as possible' instead?

File: overview.md [hunk 4]
```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
 
 Scope Frameworks limit the governance overhead by ensuring that Maker Core only focuses on high-level decisions. They also encourage innovation among SubDAOs without allowing them to take excessive risks that may damage Maker Core. 
 
-### [Decentalized Voter Committees](maker-core/dvc.md)
+### [Constitutional Voter Committees](maker-core/cvc.md)
```
Comment (LongForWisdom) [inline overview.md:53]: Probably needs more detail. In practice this is more than just a name-change. The constitutional voter committees must commit to maintaining endgame.

File: overview.md [hunk 5]
```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
...
 
-ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDAI](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDAI is one of the central products of the Endgame Plan. 
+ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDai](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDai is one of the central products of the Endgame Plan. 
+
+In the final stages of Endgame, Dai will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
 
-In the final stages of Endgame, DAI will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
+MKR tokens will no longer function as a backstop in the event that Maker is insolvent - instead, this will be reflected in the Target Price of Dai.
```
Comment (LongForWisdom) [inline overview.md:71]: Not strictly accurate. It will by default, unless governance decide it shouldn't.

File: overview.md [hunk 6]
```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
...
 
 The Endgame Plan has two major tokenomics updates. First, annual MKR minting will be used to incubate SubDAOs, fund DVCs, and incentivize the workforce. These emissions counteract the MKR burn and reduce the likelihood of MKR becoming concentrated in the hands of large holders.
 
-ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDAI](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDAI is one of the central products of the Endgame Plan. 
+ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDai](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDai is one of the central products of the Endgame Plan. 
+
+In the final stages of Endgame, Dai will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price.
```
Comment (LongForWisdom) [inline overview.md:69]: I don't think we should state this with this degree of confidence. Maybe 'Dai may not remain pegged to the US dollar. In this case, Dai may peg to another reference currency, or become free-floating, with incentive mechanisms determining its target price.' Something close to that?

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

File: overview.md (hunk 1 above)
Comment (astronautthis) [reply overview.md]: Meant to remove that sentence, done now.

**Comment by astronautthis** [comment]
@LongForWisdom comments addressed, let me know if it's good to merge!

File: overview.md [hunk 7]
```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
 
 Scope Frameworks limit the governance overhead by ensuring that Maker Core only focuses on high-level decisions. They also encourage innovation among SubDAOs without allowing them to take excessive risks that may damage Maker Core. 
 
-### [Decentalized Voter Committees](maker-core/dvc.md)
+### [Constitutional Voter Committees](maker-core/cvc.md)
 
-Decentralized Voter Committees (DVCs) are a group of voters with aligned values and views. DVCs engage in the important high-level questions of Maker Governance.
+Constitutional Voter Committees (CVCs) are a group of voters with aligned values and views. CVCs engage in the important high-level questions of Maker Governance. Every CVC must include a [Constitutional Conserver](maker-core/constitutional-conservers.md), an entity that has pledged to support the Constitution.
```
Comment (LongForWisdom) [inline overview.md:55]: "are a group of voters" -> "are groups of voters"

File: overview.md [hunk 8]
```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
...
+ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDai](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDai is one of the central products of the Endgame Plan. 
+
+In the final stages of Endgame, Dai will turn into an Unbiased World Currency. It may not remain pegged to the US dollar and may instead either peg to another reference currency, or become free-floating. In the latter case, incentive mechanisms will determine Dai's target price. 
 
-In the final stages of Endgame, DAI will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
+Maker Governance no longer guarantees that MKR tokens will function as a backstop in the event that Maker is insolvent. Governance may choose to change the Target Price of Dai to reflect this.
```
Comment (LongForWisdom) [inline overview.md:71]: Feel like you should reword this, as it's summarizing some pretty high-controversy stuff if it ever happens. Maybe:

"Maker Governance no longer guarantees that the MKR token will function as a backstop in the event that Maker becomes insolvent. Governance may instead choose to adjust the Target Price of Dai which would result in a loss in value for all Dai holders."

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**